app/Models/Asset.php

```diff
@@ -252,21 +262,11 @@ public function checkOut($target, $admin = null, $checkout_at = null, $expected_
                 $this->location_id = $target->id;
             }
         }
-        
-        /**
-         * Does the user have to confirm that they accept the asset?
-         *
-         * If so, set the acceptance-status to "pending".
-         * This value is used in the unaccepted assets reports, for example
-         * 
-         * @see https://github.com/snipe/snipe-it/issues/5772
-         */
-        if ($this->requireAcceptance() && $target instanceof User) {
-          $this->accepted = self::ACCEPTANCE_PENDING;
-        }
```

Comment:
Just confirming that this disappearing was intentional and not a result of merge weirdness.

Reply:
Yes, completely intentional. The acceptance of the asset is no longer tied to the `accepted ` property of the model. Instead a CheckoutAcceptance is generated and tied to the respective model and a user.